starter/README.md

```diff
@@ -3,21 +3,16 @@
 Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
 
 ```java
-@Component(immediate = false, service = Servlet.class, property = {
-  HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX +
-  InitParameters.SERVLET_PARAMETER_COMPATIBILITY_MODE + "=false"}) 
+@Component(immediate = false, service = Servlet.class, property = { "some.property = someValue" })
```

Comment:
I don't see _any_ point in this example.

`property = { "some.property = someValue" }` configures OSGi service parameter and nothing else.
This should be known to anyone who uses OSGi (if it's not a newbie).

The main point of this example to explain how to configure Servlet initial parameters.
That's why `HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX` has to be used in the example.
compat mode has no sense for 6.0, but some other parameter should be used then.
E.g. production mode (or there is a number of other Flow parameters )

Reply:
Got it, thanks! Does this example aim to show how to configure precisely **Vaadin** initial parameters here?

Reply:
Any servlet init parameters .
But Vaadin init parameter is the target.